installer/src/panels/com/jboss/devstudio/core/installer/DiskSpaceCheckPanel.java

```diff
@@ -66,13 +73,21 @@ private long getInstallationSize()
          Pack p = (Pack) iter.next();
          totalSize += p.nbytes;
       }
-     
+      
+      // Account for any additional features in the total aggregate size.
+      String addFeatureSizes = idata.getVariable("INSTALL_ADDITIONAL_SIZES");
```

Comment:
@pleacu I've fixed the tests and sent PR for your branch https://github.com/pleacu/jbdevstudio-product/pull/1.